src/main/java/io/github/vertxchina/TcpServerVerticle.java

```diff
@@ -14,7 +14,8 @@
 
   @Override
   public void start() throws Exception {
-    vertx.createNetServer()
+      Integer port = vertx.getOrCreateContext().config().getInteger("TcpServerVerticle.port", 32167);
```

Comment:
这里在verticle里面，已经是vert.x的context了，所以可以不用get or create context
```suggestion
      Integer port = config().getInteger("TcpServerVerticle.port", 32167);
```